draft-ietf-teep-architecture.md

```diff
@@ -664,8 +661,12 @@ that will be managing the TA in various devices. At step 3, a user
 will go to an Application Store to download the Untrusted
 Application. The Untrusted Application will trigger TA installation
 by initiating communication with a TAM. This is the step 4. The Untrusted Application
-will get messages from TAM, and interacts with device
-TEE via an Agent.
+will interact with TAM via a TEEP Broker that faciliates communications between a TAM
```

Comment:
I don't think it's ever correct to say that the "Untrusted Application will interact with a TAM".
Change "Untrusted Application will" to "TEEP Agent will".